client/src/app/feedback-list/feedback-list.component.html

```diff
@@ -1,22 +1,52 @@
-<ul *ngIf="feedbacks && feedbacks.length !== 0; else empty">
-    <li *ngFor="let feedback of feedbacks" class="flex flex-column">
-        <div [routerLink]="['/feedback']" [queryParams]="{id: feedback.id}" routerLinkActive="router-link-active"  class="flex flex-column feedback-row">
-            <p *ngIf="type === feedbackType.Received; else sent_feedbacks">
-                <strong>De:</strong> {{feedback.senderName}}
-                (<a href="mailto:{{feedback.senderEmail}}">{{feedback.senderEmail}}</a>)
-            </p>
-            <ng-template #sent_feedbacks>
-                <p>
-                    <strong>À:</strong> {{feedback.receverName}}
-                    (<a href="mailto:{{feedback.receverEmail}}">{{feedback.receverEmail}}</a>)
-                </p>
+<div class="content">
+    <table>
+        <thead>
+            <tr>
+                <th>
+                    Nom du collègue 
+                </th>
+                <th>
+                   Email du collègue
+                </th>
+                <th>
+                    Date d'envoi
+                </th>
+                <th>
+                    Consulter
+                </th>
+            </tr>    
+        </thead>
+       <tbody *ngIf="feedbacks && feedbacks.length !== 0; else empty">
+        <tr *ngFor="let feedback of feedbacks">
+            <td *ngIf="type === feedbackType.Received; else sent_name">
+                {{feedback.senderName}}
+            </td>
+            <td *ngIf="type === feedbackType.Received; else sent_email">
+                {{feedback.senderEmail}}
+            </td>
+            
+            <td>
+                {{feedback.createdAt | date: 'dd/MM/yyyy-HH:MM' }}
+            </td>
+            <td [routerLink]="['/feedback']" [queryParams]="{id: feedback.id}" routerLinkActive="router-link-active">
```

Comment:
Pourquoi tu utilise un queryParams pour l'ID, comme il est obligatoire tu pourrais le mettre directement dans le path, et ainsi avec une url de ce style /feedback/:id

```suggestion
            <td [routerLink]="['feedback', feedback.id]" routerLinkActive="router-link-active">
```